CHANGELOG.md

```diff
@@ -9,7 +9,8 @@
 
 ## Unreleased
 
-* Update to component auditing. ([PR 1996](https://github.com/alphagov/govuk_publishing_components/pull/1996)) PATCH
+* Add cookie ([PR #1999](https://github.com/alphagov/govuk_publishing_components/pull/1999)) PATCH
+* Update auditing ([PR #1996](https://github.com/alphagov/govuk_publishing_components/pull/1996)) PATCH
```

Comment:
Bit of an odd diff here?

Reply:
Just me being pernickety. I'm correcting the line that was already there - the CHANGELOG didn't match the actual PR name.